講義動画27回　課題への取り組み

KeisukeFuruta opened the pull request:
講義動画の27回をみて、Thymeleafを使用した画面描画に取り組みました。
Javaの学習をおろそかにしてしまっていたため、久々の取り組みとなります。

①studentListの画面描画
動画を見ながら実装したので、描画もできました。
<img width="924" alt="スクリーンショット 2024-12-17 17 49 48" src="https://github.com/user-attachments/assets/1316dc2f-828c-4ada-be6b-476c21597789" />

②studentCourseListの画面描画
実装できていません、Whitelabel Error Pageと表示され、エラーが発生しました。
発生エラー(1)　"courseId"のオブジェクトが見つからない→htmlファイルの<tbody>にミスがあった
<td th:text="${studentDetail.studentCourses.courseId}">1</td>
↓
<td th:text="${studentDetail.List<studentCourses>.courseId}">1</td>
この変更により、別のエラーへ派生
発生エラー(2)　After parsing a valid expression, there is still more data in the expression:

上記エラーをChat-GPTの力を借りて解決。
また、表示された列幅もCSSを使って微改善。

<img width="1461" alt="スクリーンショット 2024-12-17 20 25 20" src="https://github.com/user-attachments/assets/a05f9d0a-7dae-4a46-85b0-03e1c53d0c9f" />

KeisukeFuruta committed:
第23回課題への取り組み
studentCourseのclassを実装
studentCourseの全件検索を実装

KeisukeFuruta committed:
第23回課題への取り組み
studentCourseのclassを実装
studentCourseの全件検索を実装

KeisukeFuruta committed:
第24回課題への取り組み
30代のみの人を抽出すること
Javaコースのみを抽出すること

KeisukeFuruta committed:
第24回課題の削除

KeisukeFuruta committed:
第25回課題の取り組み
・converterの実装
・備考欄と論理削除の実装

KeisukeFuruta committed:
Update README.md

KeisukeFuruta committed:
第27回課題の取り組み
Thymeleafによる画面描画
studentListの表示

KeisukeFuruta committed:
Merge remote-tracking branch 'origin/assifnment27' into assifnment27

KeisukeFuruta committed:
第27回課題の取り組み
Thymeleafによる画面描画
studentListの表示

KeisukeFuruta committed:
第27回課題の取り組み
Thymeleafによる画面描画
studentListの表示
studentCourseListの表示へのチャレンジ

KeisukeFuruta committed:
第27回課題の取り組み
Thymeleafによる画面描画
studentListの表示
studentCourseListの表示
※ただし表示された幅が合わず、汚い

yoshi-koyama commented on README.md:
```diff
@@ -1,2 +1,4 @@
 ## StudentManagement
 Javaコースで作成中の受講生管理システムです。
+
+課題27：
```
こちらの記述ですが不完全に見えます。
書き損じですかね？

KeisukeFuruta replied:
ご確認いただきありがとうございます。
READMEの更新が途中のままプルリク出していました。
[c97f3f4](https://github.com/KeisukeFuruta/StudentManagement/pull/1/commits/c97f3f4c18f174fcc2d069f30c1ff2f4e226f07c)
こちらで更新しております、ご確認のほどよろしくお願いします。

KeisukeFuruta committed:
Update README.md

KeisukeFuruta committed:
Update README.md

KeisukeFuruta committed:
第27回課題の取り組み
Thymeleafによる画面描画
studentListの表示
studentCourseListの表示
→CSSを導入し、列幅の調整。ただしまだ綺麗にできそう

KeisukeFuruta committed:
Merge remote-tracking branch 'origin/assifnment27' into assifnment27
